internal/resources/metal/organization/resource_schema.go

```diff
@@ -0,0 +1,87 @@
+package organization
+
+import (
+	"context"
+
+	"github.com/equinix/terraform-provider-equinix/internal/framework"
+	fwtypes "github.com/equinix/terraform-provider-equinix/internal/framework/types"
+	equinix_validation "github.com/equinix/terraform-provider-equinix/internal/validation"
+	"github.com/hashicorp/terraform-plugin-framework-validators/listvalidator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+func GetResourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": framework.IDAttributeDefaultDescription(),
+			"name": schema.StringAttribute{
+				Description: "The name of the Organization",
+				Required:    true,
+			},
+			"description": schema.StringAttribute{
+				Description: "Description string",
+				Optional:    true,
+			},
+			"website": schema.StringAttribute{
+				Description: "Website link",
+				Optional:    true,
+			},
+			"twitter": schema.StringAttribute{
+				Description: "Twitter handle",
+				Optional:    true,
+			},
+			"logo": schema.StringAttribute{
+				Description: "Logo URL",
+				Optional:    true,
+			},
+			"created": schema.StringAttribute{
+				Computed: true,
+			},
+			"updated": schema.StringAttribute{
+				Computed: true,
+			},
+		},
+		Blocks: map[string]schema.Block{
+			"address": schema.ListNestedBlock{
+				Description: "Address information block",
+				Validators: []validator.List{
+					listvalidator.IsRequired(),
+					listvalidator.SizeAtLeast(1),
+					listvalidator.SizeAtMost(1),
+				},
+				CustomType: fwtypes.NewListNestedObjectTypeOf[AddressResourceModel](ctx),
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"address": schema.StringAttribute{
+							Description: "Postal address",
+							Required:    true,
+						},
+						"city": schema.StringAttribute{
+							Description: "City name",
+							Required:    true,
+						},
+						"zip_code": schema.StringAttribute{
+							Description: "Zip Code",
+							Required:    true,
+						},
+						"country": schema.StringAttribute{
+							Description: "Two letter country code (ISO 3166-1 alpha-2), e.g. US",
+							Required:    true,
+							Validators: []validator.String{
+								stringvalidator.LengthBetween(0, 18),
+								equinix_validation.StringIsCountryCode,
+							},
+						},
+						"state": schema.StringAttribute{
+							Description: "State name",
+							Optional:    true,
+							Computed:    true,
```

Comment:
It's possible the `_upgradeFromVersion` test failure is related to this line.  The SDKv2 resource only had `Optional: true` for `address.state`.  The addition of `Computed: true` means that terraform will mark the value as unknown during plan if it isn't explicitly configured.  I tried adding [a `UseStateForUnknown` plan modifier](https://developer.hashicorp.com/terraform/plugin/framework/resources/plan-modification#usestateforunknown) but that didn't seem to do anything.  Removing `Computed: true` was also not enough to resolve the issue (that may be because the ResourceModel handles null/empty `address.state` differently than the SDKv2 resource handled it.

Reply:
Oh and also had to add `stringplanmodifier.UseStateForUnknown` on the `created` and `updated` properties in the resource schema

Reply:
I pushed up a branch with my changes for comparison: 
Tests for that branch are running here: https://github.com/equinix/terraform-provider-equinix/actions/runs/8179319403

In summary:
- Removed `Computed: true` from `address.state` and updated `parseAddress` to skip setting `address.state` if it's nil or empty, as was done in SDKv2 implementation
- Marked description, website, twitter, and logo fields as `Computed: true` with a default value of `""`, which seems to match how the SDKv2 resource was represented in state

Reply:
The only test failure in CI on my branch is `TestAccMetalIPAttachment_metro`, which failed due to lack of server capacity.  How did you initially incorporate my changes into your local repo and which tests were failing locally?

Reply:
The latest commit on this PR has a bunch of failures like this:
```
    resource_test.go:67: Step 1/2 error: Error running apply: exit status 1
        
        Error: Provider produced inconsistent result after apply
        
        When applying changes to equinix_metal_organization.test, provider
        "provider[\"registry.terraform.io/hashicorp/equinix\"]" produced an
        unexpected new value: .address[0].city: was cty.StringVal("London"), but now
        null.
        
        This is a bug in the provider, which should be reported in the provider's own
        issue tracker.
        
        Error: Provider produced inconsistent result after apply
        
        When applying changes to equinix_metal_organization.test, provider
        "provider[\"registry.terraform.io/hashicorp/equinix\"]" produced an
        unexpected new value: .address[0].country: was cty.StringVal("GB"), but now
        null.
        
        This is a bug in the provider, which should be reported in the provider's own
        issue tracker.
        
        Error: Provider produced inconsistent result after apply
        
        When applying changes to equinix_metal_organization.test, provider
        "provider[\"registry.terraform.io/hashicorp/equinix\"]" produced an
        unexpected new value: .address[0].zip_code: was cty.StringVal("12345"), but
        now null.
        
        This is a bug in the provider, which should be reported in the provider's own
        issue tracker.
        
        Error: Provider produced inconsistent result after apply
        
        When applying changes to equinix_metal_organization.test, provider
        "provider[\"registry.terraform.io/hashicorp/equinix\"]" produced an
        unexpected new value: .address[0].address: was cty.StringVal("tfacc org
        street"), but now null.
        
        This is a bug in the provider, which should be reported in the provider's own
        issue tracker.
```

I recommend reverting/removing the last commit on this PR and cherry-picking the last commit from the branch I linked in a previous comment on this thread.